.gitignore

```diff
@@ -84,3 +84,6 @@ doc_tmp/
 # Config directories created by gcloud and gsutil on Jenkins
 /.config/gcloud/
 /.gsutil/
+
+# Downloaded Kubernetes source code
```

Comment:
This should say "Downloaded Kubernetes binary release"

The releases do not contain source code